index.html

```diff
@@ -51,6 +51,155 @@ <h1>I am Talha.<br>Glad to see you!</h1>
+            </div> 
+
+            <div class="card">
+                <div class="card image"><img src="card-images/3.png" alt=""></div>
+                <div class="card left-block">
+                    <div class="card head">
+                        <h2>Tonic</h2>
+                        <ul class="caption">
+                            <li class="canopy"><h3>CANOPY</h3></li>
+                            <li><img src="Counter.svg" alt="dot"></li>
+                            <li><h3>Back End Dev</h3></li>
+                            <li><img src="Counter.svg" alt="dot"></li>
+                            <li><h3>2015</h3></li>
+                        </ul>
+                    </div>
+                    
+                    <div class="card text">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <ul class="tags">
+                        <li class="tag">HTML</li>
+                        <li class="tag">CSS</li>
+                        <li class="tag">JavaScript</li>
+                    </ul>
+                    <div class="card action"><button class="see-project">See Project</button></div>
+                </div>
+            </div> 
+
+            <div class="card">
+                <div class="card image"><img src="card-images/4.png" alt=""></div>
+                <div class="card left-block">
+                    <div class="card head">
+                        <h2>Tonic</h2>
+                        <ul class="caption">
+                            <li class="canopy"><h3>CANOPY</h3></li>
+                            <li><img src="Counter.svg" alt="dot"></li>
+                            <li><h3>Back End Dev</h3></li>
+                            <li><img src="Counter.svg" alt="dot"></li>
+                            <li><h3>2015</h3></li>
+                        </ul>
+                    </div>
+                    
+                    <div class="card text">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <ul class="tags">
+                        <li class="tag">HTML</li>
+                        <li class="tag">CSS</li>
+                        <li class="tag">JavaScript</li>
+                    </ul>
+                    <div class="card action"><button class="see-project">See Project</button></div>
+                </div>
+            </div> 
+            
+        </section>
+
+        <section class="about-myself">
+            <h2>About Myself</h2>
+            <p>Hello. I’m a software developer! I can help you build a product, feature or website, look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="lets-connect">LET'S CONNECT</p>
+            <div class="icons">
```

Comment:
- [ ] the sentence **LET'S CONNECT** should use the header `hx` tag(h1, h2, h3,...). Kindly replace the `p` tag by a header ` hx` tag .